docs/docs/getting-started/02-oauth-tutorial.mdx

```diff
@@ -8,26 +8,33 @@ import gettingClientIdSecretImg from "./img/getting-started-oauth-clientid-secre
 import startAppAndSignInImg from "./img/getting-started-app-start.png"
 import githubAuthCredentials from "./img/getting-started-github-auth.png"
 import nextAuthUserLoggedIn from "./img/getting-started-nextauth-success.png"
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
 
-We know, authentication is hard. Is a rabbit hole and it's easy to get lost on it. The goal of making Auth.js is that you can add authentication easily to your project with just a few lines of code.
+We understand that authentication is tedious - It is a rabbit hole that is super easy to get lost on. The goal of making Auth.js is to help you easily add authentication to your project with just a few lines of code.
 
-The easiest way is to setup Auth.js with an [OAuth](https://en.wikipedia.org/wiki/OAuth) provider. In this tutorial we'll be setting Auth.js in a **Next.js app** to be able to login with **Github**.
+The easiest way is to set up Auth.js with an [OAuth](https://en.wikipedia.org/wiki/OAuth) provider. In this tutorial, we'll be setting Auth.js in an application with a framework of your choice to be able to log in with **GitHub**.
```

Comment:
We should have our own OAuth page under Concepts, taking the graph from https://next-auth.js.org/configuration/providers/oauth

Reply:
We have that page https://authjs.dev/concepts/oauth Are you suggesting to link it here?